util/task.py

```diff
@@ -31,6 +31,34 @@ def get_all_meter_ids(session):
            for group_meter_id in session.query(Group.group_meter_id).all()]
 
 
+def calc_timestamps():
```

Comment:
How about `calc_last_term_boundaries`? Timestamps is a bit vague.
Or `calc_reading_points_in_time` or `calc_reading_appointments`.